next/broker/celery_app/tasks.py

```diff
@@ -7,20 +7,58 @@
 import json
 import traceback
 import numpy
-from next.constants import DEBUG_ON
 import hashlib
 
-# import next.logging_client.LoggerHTTP as ell
-from next.database_client.DatabaseAPI import DatabaseAPI
-db = DatabaseAPI()
-from next.logging_client.LoggerAPI import LoggerAPI
-ell = LoggerAPI()
 import next.utils
 import next.constants
-import next.apps.Butler as Butler
+from next.constants import DEBUG_ON
+
+from next.database_client.DatabaseAPI import DatabaseAPI, DatabaseException
+from next.logging_client.LoggerAPI import LoggerAPI
+
+from next.apps.Butler import Butler
 import next.lib.pijemont.verifier as verifier
 
-Butler = Butler.Butler
+db, ell = None, None
+
+def worker_connect_db():
+    global db, ell
```

Comment:
Right, by "main script" I mean outside any functions, in global scope for this module.


``` python
db, ell = None, None

def worker_connect_db():
    # no "global db, ell"
    ...
    return db, ell

db, ell = worker_connect_db()
```

I assume `DatabaseAPI` and `LoggerAPI` return `None` if they can't connect.